api/src/test/java/org/pmiops/workbench/access/AccessTierServiceTest.java

```diff
@@ -100,44 +99,6 @@ public void test_getAllTiers_2() {
     assertThat(accessTierService.getAllTiers()).containsExactly(registeredTier, controlledTier);
   }
 
-  @Test
```

Comment:
AccessTierService.getAccessTier() no longer exists and these were simple "did you save the thing?" tests - not much value IMO